lib/curl/curl.nit

```diff
@@ -265,78 +304,88 @@ class CurlMailRequest
 	end
 
 	# Execute Mail request with settings configured through attribute
-	redef fun execute
+	fun execute: nullable CurlResponseFailed
 	do
 		if not self.curl.is_ok then return answer_failure(0, "Curl instance is not correctly initialized")
```

Comment:
This is brutal since the user is no more responsible of the initilization.

Maybe precise that the problem is not supposed to come from the nit part?